WalletWasabi/Services/WasabiSynchronizer.cs

```diff
@@ -214,13 +212,20 @@ public void Start(TimeSpan requestInterval, TimeSpan feeQueryRequestInterval, in
 									if (result.BackendCompatible && lastUsedApiVersion != WasabiClient.ApiVersion)
 									{
 										// Next request will be fine, do not throw exception.
-										return;
+										continue;
```

Comment:
Can you create a new PR with this? The rest is hack.

Reply:
Would only solve a partial problem. For example in the master Backend set to connected right after the Exception.